src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -10194,31 +10240,31 @@ void func_80847BA0(PlayState* play, Player* this) {
             invFloorPolyNormalY = 1.0f / COLPOLY_GET_NORMAL(floorPoly->normal.y);
             floorPolyNormalZ = COLPOLY_GET_NORMAL(floorPoly->normal.z);
 
-            sp4C = Math_SinS(this->yaw);
-            sp44 = Math_CosS(this->yaw);
+            sin = Math_SinS(this->yaw);
+            cos = Math_CosS(this->yaw);
 
-            this->unk_898 =
-                Math_Atan2S(1.0f, (-(floorPolyNormalX * sp4C) - (floorPolyNormalZ * sp44)) * invFloorPolyNormalY);
-            this->unk_89A =
-                Math_Atan2S(1.0f, (-(floorPolyNormalX * sp44) - (floorPolyNormalZ * sp4C)) * invFloorPolyNormalY);
+            this->floorPitch =
+                Math_Atan2S(1.0f, (-(floorPolyNormalX * sin) - (floorPolyNormalZ * cos)) * invFloorPolyNormalY);
+            this->floorPitchAlt =
+                Math_Atan2S(1.0f, (-(floorPolyNormalX * cos) - (floorPolyNormalZ * sin)) * invFloorPolyNormalY);
 
-            sp4C = Math_SinS(this->actor.shape.rot.y);
-            sp44 = Math_CosS(this->actor.shape.rot.y);
+            sin = Math_SinS(this->actor.shape.rot.y);
+            cos = Math_CosS(this->actor.shape.rot.y);
 
-            D_80853610 =
-                Math_Atan2S(1.0f, (-(floorPolyNormalX * sp4C) - (floorPolyNormalZ * sp44)) * invFloorPolyNormalY);
+            sFloorPitchShape =
+                Math_Atan2S(1.0f, (-(floorPolyNormalX * sin) - (floorPolyNormalZ * cos)) * invFloorPolyNormalY);
```

Comment:
Maybe `sShapeFloorPitch` or `sFloorShapePitch` to be more consistent with `ShapeYaw`?

Reply:
Both of those read incredibly off to me, while with yaw it sounds fine. I dont know if I can really articulate why
```
static s32 sShapeYawToTouchedWall = 0;
static s32 sWorldYawToTouchedWall = 0;
static s16 sFloorPitchShape = 0;
```
with yaw, it is describing the type of generic yaw, so its a "shape yaw" and "world yaw", while with pitch, its already being described as the "floor pitch", with shape being added as an extra descriptor at the end. 

I suppose `sYawToTouchedWallWorld` and `sYawToTouchedWallShape` arent terrible, and is the only way I'd be willing to create consistency. But I also dont think consistency is crucial here as they are separate things

Reply:
I think it's more the `Shape` at the the end of the name that's throwing me off, and less about the consistency. Like reading that I think it's a "Shape" of some sorts that's modified depending on the pitch, but really it's the pitch of the slope directed in player's "shape-yaw". Something like `sFloorPitchTowardsShape` but that doesn't really work either.

Maybe phrased a different way, it feels strange that the main part "pitch" is the middle of the variable `sFloorPitchShape`, and the adjectives/modifiers are on the ends. Does that make sense? But nothing above really stands out so idk

Reply:
its meant to be read as "floor pitch (shape)", which is why its at the end. its adding more context that this one is the shape version of the already existing `floorPitch` (instance variable) which is implied to be world 

I think its hard to read it as a literal shape when you have the context that its related to angles, which always involve world and shape. but maybe there is still a better way to phrase it

Reply:
fwiw I read this name as "shape of floor pitch" as well even though it doesn't make sense to, since I'm more familiar with seeing the qualifier (shape/world) before the angle (yaw/pitch/roll). I'm partial to `sFloorShapePitch` since it just fits better with my existing notions about these names

Reply:
I dont agree that it sounds better but will concede, pushing change